src/posix/README.md

```diff
@@ -0,0 +1,84 @@
+OpenThread POSIX app
+====================
+
+OpenThread supports running its core on POSIX and transmits radio frames through a radio transceiver.
+
+Currently most platforms in [examples/platforms](../../examples/platforms) support transceiver mode.
+
+The figure below shows the architecture of OpenThread running in transceiver mode.
+
+```
++-------------------------+
+|     MLE TMF UDP IP      |
+|  MeshForwarder 6LoWPAN  |
+| _ _ _ _ _ _ _ _ _ _ _ _ |      spinel         +------------------+
+|    OpenThread Core      | <---------------->  | OpenThread Radio |
++-------------------------+     UART|SPI        +------------------+
+         POSIX                                          Chip
+```
+
+Build
+-----
+
+```sh
+# build core for POSIX
+make -f src/posix/Makefile-posix
+# build transceiver, where xxxx is the platform name, such as cc2538, nrf52840 and so on
+make -f examples/Makefile-xxxx
+```
+
+Test
+----
+
+**NOTE** Assuming the build system is 64bit Linux, you can play just like the normal OpenThread as described in the [command line document](../../src/cli/README.md).
```

Comment:
I would update to 

"...you can **use the normal OpenThread CLI** as described in..."